We do not need to check docs, coverage, typing for this project.

--- portend-3.2.1/tox.ini.orig
+++ portend-3.2.1/tox.ini
@@ -8,10 +8,7 @@
 usedevelop = True
 extras =
 	test
-	check
-	cover
 	enabler
-	type
 
 [testenv:diffcov]
 description = run tests and check that diff from main is covered
